### DISCUSSION TITLE
Commits en caso de problemas para avanzar con el código

### DISCUSSION THREAD

**Discussion by perealberto** [MarioArocaPaez/decide#8] [Q&A] [answered]
¡Buenas a todos! Este pequeño hilo de discusión lo vamos a utilizar para inaugurar el canal de discusiones del repositorio. 😃 

Contexto: @silvglad y yo estamos desarrollando la _Feat: Pregunta SI/NO (#1)_. @silvglad  se está encargando de la modificación de la clase _Question_ añadiendo un atributo (más info. en la descripción de la feature) y esto ha hecho que se produzca un problema de compatibilidad con algunas vistas y los modelos.

Problema/Pregunta: No sabemos si estaría bien que @silvglad hiciese commit, aunque esté dando fallos, para que yo también pudiese ver el código y en pareja buscar el problema, o, por el contrario, evitar esta práctica de subir los cambios que producen error (a la rama de la feature) aunque sea para intentar solucionarlo. Otra posibilidad es esperarnos a poder verlo en persona.

Así que, de cara a este y futuros problemas, ¿qué hacemos?

@laurolmer @MarioArocaPaez @claugp06

**Comment by MarioArocaPaez** [comment, marked as answer]
Yo creo que con que os quedéis en vuestra rama tiene sentido hacerlo. Al fin y al cabo, tenemos que trabajar en equipo. Lo que quizás convendría sería añadir en el cuerpo del mensaje de commit que está dando fallos y se está subiendo con el propósito de ser revisado por otros compañeros. Por si en algún momento hacemos rollback.
Eso sí, hasta que no estemos 100% seguros de que todo está bien no se mergea con develop, que no me entere yo :rage:

**Reply by claugp06** [reply]
Opino lo mismo y me parece buena la idea de indicar en el commit que da fallos

**Reply by laurolmer** [reply]
Mientras no pase a develop con el fallo, me parece correcto como han dicho los compañeros

**Reply by perealberto** [reply]
Okey, me parece perfecto. Si estamos todos de acuerdo, pues pa'lante! :smiley:
